src/comunicationsmanagerfilesockets.cpp

```diff
@@ -134,15 +134,13 @@ int ComunicationsManagerFileSockets::create_new_socket(int *sockId)
 ComunicationsManagerFileSockets::ComunicationsManagerFileSockets()
 {
     count = 0;
-    mtx = new MegaMutex();
-    informerMutex = new MegaMutex(false);
+    mtx = new std::mutex();
+    informerMutex = new std::mutex();
```

Comment:
great to see std::mutex getting used - I think these two can just be normal class members now, rather than pointers that we need to new/delete?  thanks